services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/assets/index.js

```diff
@@ -0,0 +1,234 @@
+import PageHeader from '@/components/page-header'
+import TypeTag from '@/components/type-tag'
+import { framework } from '@/data/framework'
+import { ALPHABETICAL_ORDER, sortItems } from '@/data/sort'
+import { getLibraryData } from '@/lib/github'
+import pageStyles from '@/pages/pages.module.scss'
+import { assetSortComparator } from '@/utils/schema'
+import { getSlug } from '@/utils/slug'
+import { mediaQueries, useMatchMedia } from '@/utils/use-match-media'
+
+import styles from './index.module.scss'
+
+const headerData = [
+  {
+    key: 'name',
+    header: 'Asset'
+  },
+  {
+    key: 'type',
+    header: 'Type'
+  },
+  {
+    key: 'tags',
+    header: 'Tags'
+  },
+  {
+    key: 'status',
+    header: 'Status'
+  },
+  {
+    key: 'link',
+    header: ''
+  }
+]
+
+const LibrayAssets = ({ libraryData, params }) => {
+  const isLg = useMatchMedia(mediaQueries.lg)
+
+  const [sort, setSort] = useState(ALPHABETICAL_ORDER)
+  const router = useRouter()
+
+  if (router.isFallback) {
+    return (
+      <Grid>
+        <Column sm={4} md={8} lg={16}>
+          <div className={pageStyles.content}>
+            <h1>Loading...</h1>
+          </div>
+        </Column>
+      </Grid>
+    )
+  }
+
+  const { description } = libraryData.content
+
+  const seo = {
+    title: 'Library assets',
+    description
+  }
+
+  const assets =
+    libraryData.assets?.sort(assetSortComparator(sort)).map((asset) => {
+      const assetRow = {
+        id: asset.content.id,
+        name: <span className={styles.assetName}>{asset.content.name}</span>,
+        type: <TypeTag type={asset.content.type} className={styles.tag} />,
+        status: <CatalogItemMeta asset={asset} properties={['status']} />,
+        tags: (
+          <span className={styles.truncatedText}>
+            {asset.content.tags.join('; ').replaceAll('-', '‑')}
+          </span>
+        ),
+        link: (
+          <Link href={`/assets/${asset.params.library}/${params.ref}/${getSlug(asset.content)}`}>
+            <a className={styles.rowAnchor}>
+              <ArrowRight size={16} className={styles.arrowIcon} />
+            </a>
+          </Link>
+        )
+      }
+      if (asset.content.framework === framework['design-only']) {
+        assetRow.type = (
+          <div style={{ display: 'flex' }}>
+            <TypeTag type={asset.content.type} className={styles.tag} />
+            <TypeTag type={'design-only'} className={clsx(styles.designTag, styles.tag)} />
+          </div>
+        )
+      }
+      return assetRow
+    }) ?? []
+
+  const onSort = (sortOrder) => setSort(sortOrder)
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <Grid className={styles.libraryAssetsContainer}>
+        <Column sm={4} md={8} lg={{ start: 5, span: 12 }}>
+          <PageHeader title={seo.title} />
+        </Column>
+        <Column sm={4} md={8} lg={4}>
+          {/* In page nav  */}
+        </Column>
+        <Column sm={4} md={8} lg={12}>
```

Comment:
```suggestion
        <Column sm={4} md={8} lg={{ start: 5, span: 12 }}>
```

I think we can just do this here instead, the library side nav will be controlled from the layout.